ios/jailed/Makefile

```diff
@@ -0,0 +1,15 @@
+# Don't modify these three variables
+DEBUG=0
+TARGET_CODESIGN=
+override _LOCAL_USE_SUBSTRATE=
```

Comment:
Sure. Should the makefile include tweak.mk after overriding everything, or should I copy the code from tweak.mk into tweak_jailed.mk?